.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
 - repo: https://github.com/antonbabenko/pre-commit-terraform
-  rev: v1.39.0
+  rev: v1.89.0
```

Comment:
needed to bump the version for hooks to be compatible with latest `brew install terraform-docs`